spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe "the customer show" do
+  it "shows customer and items" do
+    customer = Customer.create(name:"Bob")
+
+    supermarket_1 = Supermarket.create(name:"Safeway", location:"Denver")
+    item_1 = Item.create(name: "Peas", price: 2, supermarket_id: supermarket_1.id)
+    item_2 = Item.create(name: "Carrots", price: 2, supermarket_id: supermarket_1.id)
+    item_3 = Item.create(name: "Chicken", price: 5, supermarket_id: supermarket_1.id)
+
+    customer.add_item(item_1.id)
+    customer.add_item(item_2.id)
+    customer.add_item(item_3.id)
+
+    visit ("/customers/#{customer.id}") 
+
+    expect(page).to have_content(customer.name)
+    expect(page).to have_content(item_1.name)
```

Comment:
Remember that we can use `within` blocks to help test more robustly where things are on our page.